client/src/components/App.jsx

```diff
@@ -74,7 +74,7 @@ class App extends React.Component {
     const { reviews, rating, aggregateRating } = this.state;
     return (
       <div className={classNames({ [styles.body]: true, container: true })}>
```

Comment:
@gorakong Why are you using the classnames package?

Reply:
@jcelder  I used it to allow for the usage of multiple classes (i.e., for bootstrap classes & my own css)

Reply:
@gorakong I see. I don't believe you need to do that with CSS modules. You can import your classes into an object and then refer to them as keys on that object. For example if you wanted to add classes from bootstrap and classes from your own css you could potentially do the following.
```
import styles from './styles.css'
import bootstrap from '/path/to/bootstrap.css'

<div
  className={[styles.coolStyle, bootstrap.btn].join(' ')}
>
</div>
```

I am going to approve this PR for now, but I would strongly recommend refactoring your class names to use CSS modules.

Reply:
Sounds good! I will be making this refactor.